Fix a few issues and styles

jazeee opened the pull request:
Fix Overview tab, Skill mod column is too narrow for double digits. #62

Fix Consider making grid columns use `word-break: break-word` #61

Fix Small formatting issues #60

* Innate spells, update column title to `Damage/Duration`.
* For arcane spells, adjust columns:
  * the mod column should not stretc
  * the target column should stretch.
  * Also, adjusted range and target to be stretch.
* Fix button hover styles.
* Move Notes popover icon to be next to name
* For innate, make name be bold and centered when starting with dashes.


## Screenies

| Old | New |
| ---- | ---- |
| ![image](https://user-images.githubusercontent.com/5629800/212509513-ce183fb8-a94e-441e-bfcb-f050e6e2cf4c.png) | ![image](https://user-images.githubusercontent.com/5629800/212510259-7295f506-3298-4801-97d1-87b24c5f49d3.png) |

Others:
![image](https://user-images.githubusercontent.com/5629800/212770850-56cf3701-0e7a-4171-bfc1-b6777d0758bd.png)

![image](https://user-images.githubusercontent.com/5629800/212770830-12b7bd72-8563-40ad-9b16-212b1be062f9.png)
![image](https://user-images.githubusercontent.com/5629800/212779276-c2fd34f5-04c7-40c0-a7b5-151e957c75b9.png)

Button examples for spells with or without notes.
![image](https://user-images.githubusercontent.com/5629800/212784590-6192addb-f226-43f5-8b3a-94b74736e4cb.png)

Commit:
Fix a few issues and styles

Fix Overview tab, Skill mod column is too narrow for double digits. #62

Fix Consider making grid columns use `word-break: break-word` #61

Fix Small formatting issues #60

* Innate spells, update column title to `Damage/Duration`.
* For arcane spells, adjust columns:
  * the mod column should not stretc
  * the target column should stretch.
  * Also, adjusted range and target to be stretch.
* Fix button hover styles.
* Move Notes popover icon to be next to name
* For innate, make name be bold and centered when starting with dashes.

Commit:
Adjust a few more styles

Commit:
Adjust grid word-break

Commit:
Clean up some legacy css selectors #54

jazeee commented on bin/main.html:
```diff
@@ -939,27 +939,16 @@ <h2>
-      </select>
-      <input class="spellrange-grid" name="attr_spellrange" type="text" />
-      <input class="spelltarget-grid" name="attr_spelltarget" type="text" />
-      <input class="spellduration-grid" name="attr_spellduration" type="text" />
```
I rearranged to move the popover next to the name.

jazeee commented on ui/basicTab/basicTab.pug:
```diff
@@ -2,7 +2,7 @@ include ../global/advantages.pug
 // Basic section 
 .basic.section.basic-tab
-  .advantages-and-feats.horizontal-flex
+  .advantages-and-feats
```
As a project grows larger, it's generally better to name classes by concept, not by style, and style them specifically.

jazeee commented on ui/buttons/buttons.scss:
```diff
@@ -12,9 +12,18 @@ button.actionRollButton {
+    background-image: unset;
+    background-color: $background-color;
+  }
+}
```
Fixes the white hover on buttons, and cleans up.

johnlamping commented:
Nice.

One small thing: the attribute column should also be fixed width. It just needs to be wide enough to hold "IQ+DX".

Two larger things, maybe for new issues:
   Using the initial "-" to change formatting doesn't make it obvious to the user that they have that option. It might be better to add a column, like the Discipline? column, that says whether a row is the basic innate casting ability for a sphere. It could be titled something like "Sphere?", and have two values, "S", or " ".
   One disadvantage of the note popovers is that there is no way to tell if there is a note, without clicking to see. It would be nice if the styling of the magnifying class changed if there was a note to be found there.

jazeee commented:
Fixed the attribute width.
![image](https://user-images.githubusercontent.com/5629800/212768598-bbfd1c0b-7a8d-4b1f-bf61-826e28fb29f3.png)

Updated to add a `S` option and style based on that.

![image](https://user-images.githubusercontent.com/5629800/212768562-ab220a48-aae2-4506-8296-58527be1795e.png)

I'll work on the note button as a separate ticket. I also felt it needs a bit more.
https://github.com/epic-power-rpg/roll20/issues/67

Commit:
Fix some styles and adjust the UI

jazeee commented:
Updated.

Commit:
Add tool tip for Sphere

jazeee commented:
@johnlamping feel free to review last commit that adds the styling changes and `Sphere?` column.

I also added a tooltip for this long column:
![image](https://user-images.githubusercontent.com/5629800/212779276-c2fd34f5-04c7-40c0-a7b5-151e957c75b9.png)

Commit:
Fix Adjust the note popover icon #67

Change to be different icon content and color, depending on note value.

jazeee commented:
Also reworked the notes button to look like:
![image](https://user-images.githubusercontent.com/5629800/212784590-6192addb-f226-43f5-8b3a-94b74736e4cb.png)

johnlamping reviewed (approved):
👍!
